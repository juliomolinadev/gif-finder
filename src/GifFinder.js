import React from "react";

export const GifFinder = () => {
	return (
		<div>
			<h1>Gif Finder</h1>
		</div>
	);
};
