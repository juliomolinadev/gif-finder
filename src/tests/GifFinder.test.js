import React from "react";
import { shallow } from "enzyme";
import { GifFinder } from "../GifFinder";

describe("Tests in <GifFinder /> ", () => {
	test("should be displayed correctly", () => {
		const wrapper = shallow(<GifFinder />);
		expect(wrapper).toMatchSnapshot();
	});
});
