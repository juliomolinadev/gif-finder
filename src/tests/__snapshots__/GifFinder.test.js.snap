// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <GifFinder />  should be displayed correctly 1`] = `
<div
  className="container text-secondary"
>
  <div
    className="d-flex justify-content-center mt-5"
  >
    <h1>
      Gif Finder
    </h1>
  </div>
  <div
    className="row d-flex justify-content-center mt-4"
  >
    <div
      className="col-md-6"
    >
      <SearchBar
        setWord={[Function]}
      />
    </div>
  </div>
  <div
    className="mt-4 animate__animated animate__fadeIn"
  >
    <GifGrid
      word=""
    />
  </div>
</div>
`;
